cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -261,7 +261,7 @@ func (s *Server) Run() {
 		// We update the list of indexed repos every Interval. To speed up manual
 		// testing we also listen for SIGUSR1 to trigger updates.
 		//
-		// "pkill -SIGUSR1 zoekt-sourcegra"
+		// "pkill -SIGUSR1 zoekt-sourcegraph"
```

Comment:
```suggestion
		// "pkill -SIGUSR1 zoekt-sourcegra"
```

pkill has a limit on how much of the command name it matches. Yay old unix things

Reply:
🤦